### PR TITLE
[RI2] Update README for Lakelse

### LINKED ISSUE #2731: [RI2] Update release information in README.md
Update the release information in the readme file.

### PR DESCRIPTION
**Pull request by rihabbanday**
Fixes #2731

### CODE REVIEW

**Review by pgoyal01 — APPROVED** [review]

**Review by michaelspedersen — APPROVED** [review]
LGTM

**Review by acmacm — APPROVED** [review]

**Comment by pgoyal01** [comment]
@walterkozlowski Can you please merge and approve?

File: doc/ref_impl/cntt-ri2/README.md [hunk 1]
```diff
@@ -5,19 +5,20 @@
 This is the Kubernetes-based Reference Implementation (RI-2)
 
 ## Release Information
-**Bundle: _6.0-alpha_**
+**Bundle: _6.0_**
```
Comment (gkunz) [inline doc/ref_impl/cntt-ri2/README.md:8]: Sorry for missing the call. I assume you discussed whether to go for 6.0 or 7.0? We switched from 4.0-alpha to 6.0-alpha as well without a 4.0 release. Just curious.

Comment (pgoyal01) [reply doc/ref_impl/cntt-ri2/README.md:8]: @gkunz Yes we discussed and as the RA2 LAkelse was labeled 7.0 and RI2 is a version behind we settled on 6.0

Comment (gkunz) [reply doc/ref_impl/cntt-ri2/README.md:8]: sounds good to me.

**Review by gkunz — APPROVED** [review]

**Pull request merged**